### ISSUE TITLE
Tagged logger is broken in Rails. 

### ISSUE THREAD

**Issue by ulversson** [anycable/anycable-rails#4]
I am getting this 
```ruby
undefined method `add_tags' for #<ActiveSupport::Logger:0x000000047b0940>
```

when using tagged loger in application_cable/connection.rb file
When this is commented out like this

```ruby

    def connect
      self.current_user = find_verified_user
      #logger.add_tags 'ActionCable', current_user.email
    end
````
problem disappears and I am able to connect without any problems. 
(Getting "Auth failed" with tagged logger from anycable-go websocket server)

**Comment by klacointe** [comment]
Same issue here.

**Comment by palkan** [comment]
Fixed in 0.4.5.

The fix, actually, just ignores the usage of tags when running in AnyCable ('cause we do not have _persistent_ loggers).
There can be a workaround though. For example, we can provide a special method to configure the logger (i.e. extract this functionality from `#connect`).
